File: csharp/SlimDownYourAggregates.Tests/Original/Entities/CommitteeApproval.cs
namespace SlimDownYourAggregates.Tests.Original.Entities;

public class CommitteeApproval
{
    public CommitteeApproval(bool isApproved, string feedback)
    {
        IsApproved = isApproved;
        Feedback = feedback;
    }

    public bool IsApproved { get; }
    public string Feedback { get; }
}

File: java/src/main/java/io/eventdriven/slimdownaggregates/original/domain/books/entities/CommitteeApproval.java
package io.eventdriven.slimdownaggregates.original.domain.books.entities;

import io.eventdriven.slimdownaggregates.original.infrastructure.valueobjects.NonEmptyString;

import static io.eventdriven.slimdownaggregates.original.infrastructure.validation.Validation.assertNotNull;

public record CommitteeApproval(boolean isApproved, NonEmptyString feedback) {
  public CommitteeApproval{
    assertNotNull(feedback);
  }
}
